docs/00_what_is_cvmfs.md

```diff
@@ -1,13 +1,147 @@
+triggered when search paths are examined.
+
+### De-duplication of files
+
+Due to the content-addressable storage that is used under the hood, CernVM-FS automatically
+effectively stores the contents of a file only once, even when it is included multiple times
+in a particular repository at different paths.
+
+This can result in a significant reduction in storage capacity that is required to host a large stack of
+many software installations, especially when identical files are spread out across the repository,
+as often happens with particular files like example data files across multiple versions of the same software.
+
+### Compression of data
+
+CernVM-FS [stores file content with compression](https://cvmfs.readthedocs.io/en/stable/cpt-repo.html#compression-and-hash-algorithms),
+on the server, which not only further reduces required storage space but also significantly limits the
+network bandwidth that is used to serve that a stored files in a repository across the network.
+On the client side, the data is transparently decompressed as a part of the POSIX-compatible way in which
+the contents of a CernVM-FS repository is presented under `/cvmfs`.
+
+### Automatic updating of repository contents
+
+...
+
+### Multi-level caching
+
+To reduce the latency when accessing files in a CernVM-FS repository, a multi-level caching hierarchy
+is employed.
+
+First of all, CernVM-FS benefits from the standard Linux kernel file system cache on a client system:
+the content of a file that have been read recently will already be cached in memory by the kernel,
+so that file does not need to be read from disk again.
+
+CernVM-FS also maintains a custom [local cache](https://cvmfs.readthedocs.io/en/stable/cpt-details.html#disk-cache)
+on the client system with a configurable size, which uses a
+[least-recently used (LRU)](https://en.wikipedia.org/wiki/Cache_replacement_policies#LRU) replacement policy.
+Files in a CernVM-FS repository that have been accessed sufficiently recently on a particular system
+will already be available in the local cache of that system, and do not need to be downloaded again from
+the network of CernVM-FS servers that support that repository.
+
+When the contents of a file that is being accessed is not available yet in the local client cache,
+CernVM-FS will automatically (and transparently) download it either from a [forward proxy server](#squid-proxy)
+(like [Squid](http://www.squid-cache.org/)), or from a [Stratum-1 replica server](#stratum-1-replica-server).
+Both the proxy and the replica server could be within the same local network as the client, or not.
+To help reduce performance problems regarding network latency and bandwidth, clients can leverage
+the [Geo API](https://cvmfs.readthedocs.io/en/stable/cpt-replica.html#geo-api-setup) supported by
+Stratum-1 replica servers to automatically sort them geographically,
+in order to prioritize connecting to the closest ones.
```

Comment:
It may make sense to denote these caching levels as "site-level" / "cluster-level" cache and global mirror servers.